### ISSUE TITLE
WYSIWYG field not picking up last used visual vs text setting, not loading in text can change some HTML

### ISSUE THREAD

**Issue by dweuste** [pods-framework/pods#6176]
In the WYSIWYG field, there are times you have to use the Text Editor rather than the Visual Editor. It is still superior to the standard code editor because clients can still highlight and bold and italicize and link and not have to worry about paragraph tags, but when the editor switches to Visual, it breaks the code if there were some non-standard things. An example of this is adding text in between list items (text <li></li> text <li></li>). Normally this isn't an issue because once the editor is in Text mode, it doesn't switch unless you tell it to. But after 2.8, that's no longer the case, now after each page refresh, it reverts to Visual Editor and breaks that previous code, so you have to re-do whatever you previously did every single time you want to edit. This is mainly an issue in WooCommerce pages if that matters.

**Comment by JoryHogeveen** [comment]
Yes there is. We could add two fields to the Editor settings: `tinymce` and `quicktags` (both enabled by default) and let users decide what to do.
See available settings in: https://developer.wordpress.org/reference/classes/_wp_editors/parse_settings/

**Comment by sc0ttkclark** [comment]
I've got an initial fix for the setting of the default editor on the page -- however it's not running at the right time to switch the editors.

Code is in this change: 7f0aa0708ef41c08ef8a7235478a56fd66a1fe12 (and subsequent commits on 2.8.1)

**Comment by sc0ttkclark** [comment]
I'm removing the defaultEditor handling for now, the solution would still present the problem reported since it loads as visual editor and then switches to the HTML editor (causing the HTML mess up).

I suggest using the Code field type for now @dweuste

**Comment by dweuste** [comment]
A question as this is still being worked on. Is there a way to make the code editor add or interpret line breaks? Getting clients to wrap all text in spans hasn't been super consistent, and years old code end up doing some strange things without them.
